### ISSUE TITLE
Doc search is not global

### ISSUE THREAD

**Issue by Mariangela** [wso2/docs-apim#838]
**Description:**
The newly added doc search that is powered by Algolia not global. This needs to be fixed because otherwise, the search restricts the users to one doc space.

**Comment by tmkasun** [comment]
Need to set site metadata as instructed in [Algolia Docs](https://docsearch.algolia.com/docs/required-configuration/#introduce-global-information-as-meta-tags)

**Comment by shaniR** [comment]
I don't think we need to have a global search, across versions. @Mariangela  what is the usecase we are trying to address?
 IMO, a user will select the version and then search withing the version.

**Comment by tmkasun** [comment]
Sent two PRs[1][2] for master and 3.0.0 branches,  We need to make sure that we update these meta tags whenever a new version is released

[1]: https://github.com/wso2/docs-apim/pull/856
[2]: https://github.com/wso2/docs-apim/pull/857

**Comment by Mariangela** [comment]
@shaniR , sorry, ya that was what I meant but it came out wrong. What I meant was that the search should be relevant to the version that you are in.

**Comment by tmkasun** [comment]
Reopening the issue, As we need to use [facet filters](https://www.algolia.com/doc/guides/managing-results/refine-results/filtering/#facet-filters) to filter out the search result based on the meta tags we have put for each version of the docs

**Comment by tmkasun** [comment]
I think we still have to fix the problem of updating the algolia config per every release.For example

![image](https://user-images.githubusercontent.com/3313885/99044683-292ca780-25b6-11eb-94f1-a8afef8b2db1.png)

when we release apim 4.0 we need update above config as 

```
  "sitemap_urls": [
    "https://apim.docs.wso2.com/en/latest/sitemap.xml", // 4.0.0
    "https://apim.docs.wso2.com/en/next/sitemap.xml", // 4.1.0
    "https://apim.docs.wso2.com/en/3.0.0/sitemap.xml",
    "https://apim.docs.wso2.com/en/3.1.0/sitemap.xml"
    "https://apim.docs.wso2.com/en/3.2.0/sitemap.xml"

  ],
```

This can be avoided by putting a consolidated sitemap file in the algolia config and update the consolidated sitemap per release from our side.

@shaniR WDYT ?

**Comment by shaniR** [comment]
yes let's do it.
